packages/amp/src/__tests__/processors.test.tsx

```diff
@@ -0,0 +1,219 @@
+import Html2React from "@frontity/html2react/src/libraries/component";
+import { HelmetProvider } from "frontity";
+import { amp, toBeValidAmpHtml } from "./__utilities__/amp-validator";
+import { FilledContext, HelmetData } from "react-helmet-async";
+
+import processors from "../processors";
+import { render } from "@testing-library/react";
+
+expect.extend({ toBeValidAmpHtml });
+
+// Need to set that flag when testing with jest
+// https://github.com/staylor/react-helmet-async#usage-in-jest
+HelmetProvider.canUseDOM = false;
+
+const replaceHeadAttributes = (head: HelmetData) =>
+  head.script.toString().replace(/async=("|')?true("|')?/, "async");
```

Comment:
The AMP validator is what Google runs to validate if an AMP page is valid or not, so unfortunately we have to use `async=""`.

@darerodz also found this problem on his PR.

```suggestion
const replaceHeadAttributes = (head: HelmetData) => head.script.toString();
```

But it seems like your code is already fine with the `amp={undefined}` trick, because if I remove that replace the validator passes fine 🙂